src/server/index.jsx

```diff
@@ -6,6 +6,8 @@ import HomePage from './pages/home-page'
 import LoginPage from './pages/login-page'
 import UserPage from './pages/user-page'
 
+import BaseStyle from './style/base' // eslint-disable-line
```

Comment:
There is likely a better way of doing this with emotion's `injectGlobal` that doesn't require an `eslint-disable-line`

Reply:
just curious, does `import './style/base'` work -- along the lines of PR #882, sometimes you can just run the code you need by invoking the import without actually pulling in any variable